taf/updater/updater.py

```diff
@@ -541,9 +545,21 @@ def _update_named_repository(
                         )
                         if error:
                             raise error
+                        return error
                     except Exception as e:
                         errors.append(str(e))
```

Comment:
Since we can no longer append the error to the errors list here, we should return here